plugins/modules/keycloak_realm.py

```diff
@@ -870,6 +877,7 @@ def main():
         ),
         admin_events_details_enabled=dict(type="bool", aliases=["adminEventsDetailsEnabled"]),
         admin_events_enabled=dict(type="bool", aliases=["adminEventsEnabled"]),
+        admin_permissions_enabled=dict(type='bool', aliases=['adminPermissionsEnabled']),
```

Comment:
The formatter likely wants you to change `'` to `"`:
```suggestion
        admin_permissions_enabled=dict(type="bool", aliases=["adminPermissionsEnabled"]),
```